SAGE Journals.js

```diff
@@ -198,6 +198,10 @@ function scrape(doc, url) {
 					if (tags) item.tags = tags.map(n => n.textContent);
 				}
 			}
+			// mark articles as "LF" (MARC=856 |z|kostenfrei), that are published as open access
+			let openAccessTag = doc.querySelector('.accessIcon');Z.debug(openAccessTag)
+			let altLabel = openAccessTag.alt;Z.debug(altLabel)
+			if (openAccessTag && altLabel.match(/open\saccess/gi)) item.notes.push({note: 'LF:'});
```

Comment:
Please remove the debug statements. 
Access to attributes should always make make sure that the selected element exists beforehand, so
```javascript
let openAccessTag = doc.querySelector('.accessIcon');
let altLabel = openAccessTag && openAccessTag.hasAttribute("alt") ? openAccessTag.alt : "";
if (altLabel.match(/open\s+access/gi) ...
```
Alternatively, and maybe more concise would be (_beware_: naming applies to T&F) 
```javascript
let AccessIconLocation = doc.querySelector('.accessIconLocation[alt]');
if (AccessIconLocation && AccessIconLocation.alt.match(/open\s+access/gi))
     item.notes.push('LF:');
```

Reply:
Thanks for the code review. I choose the alternative solution.